.github/workflows/main.yaml

```diff
@@ -11,6 +11,8 @@ jobs:
         - { target: "x86_64-unknown-linux-musl", install_openssl_args: "linux-x86_64 x86_64-linux-musl-"}
         - { target: "armv7-unknown-linux-gnueabihf", install_openssl_args: "linux-armv4 arm-linux-gnueabihf-"}
         - { target: "armv7-unknown-linux-musleabihf", install_openssl_args: "linux-armv4 arm-linux-musleabihf-"}
+        - { target: "aarch64-unknown-linux-gnu", install_openssl_args: "linux-aarch64 aarch64-unknown-linux-gnu"}
+        - { target: "aarch64-unknown-linux-musl", install_openssl_args: "linux-aarch64 aarch64-unknown-linux-musl"}
```

Comment:
The openssl args should be `linux-aarch64 aarch64-linux-musl-`. [See here](https://github.com/rust-embedded/cross/blob/a7b430e578642968498989c629c08303f387cb4e/docker/Dockerfile.aarch64-unknown-linux-musl#L30)